Makefile

```diff
@@ -70,10 +72,13 @@ antctl-instr-binary:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) test -tags testbincover -covermode count -coverpkg=antrea.io/antrea/pkg/... -c -o $(BINDIR)/antctl-coverage $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+# diable cgo for antrea-cni since it can be installed on some systems with
+# incompatible or missing system libraries.
 .PHONY: windows-bin
 windows-bin:
 	@mkdir -p $(BINDIR)
-	GOOS=windows $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/antrea-cni antrea.io/antrea/cmd/antrea-agent
+	GOOS=windows CGO_ENABLED=0 $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/antrea-cni
+	GOOS=windows $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/antrea-agent
```

Comment:
Doesn't antrea-agent run as a process on windows? Should it also have cgo disabled, assuming it could be complied on windows OS (so no cross-compiling)?

Reply:
Thanks for the reminder, you are correct. In the case of the `antrea/antrea-windows` Docker image, we actually build the binaries without cross-compilation. Then when running on Windows in the "Docker mode" (as opposed to "containerd mode"...) we copy the antrea-agent binary to the host and run it as a host process: https://github.com/antrea-io/antrea/blob/971d66c99b4fbdd962bfe4fc3e5d2c623933df14/build/yamls/antrea-windows.yml#L7. So it's definitely a valid assumption.